examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml

```diff
@@ -142,7 +142,7 @@
                             jcr:primaryType="nt:unstructured"
                             jcr:title="formcontainer-policy"
                             sling:resourceType="wcm/core/components/policy/policy"
-                            components="[group:Core Components Examples - Adaptive Form,group:Core Components Examples - Forms And Communications Portal,group:Core Components Examples]">
+                            components="[group:Core Components Examples - Adaptive Form,group:Core Components Examples - Forms And Communications Portal,group:Core Components Examples,group:replace test group]">
```

Comment:
Please don't change examples for test case, ideally any test case related changes should be in the collateral present as part of `it` folder, since examples is delivered to aemcomponents.dev

Reply:
Would it be fine if using cypress, we just update the template checking the 'replace test group'. This way, it will not be in code.